File: 01 - Applications/api/src/main/java/ch/futnetsinglemaster/api/beans/ResultJSON.java
package ch.futnetsinglemaster.api.beans;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ResultJSON {

    private int responseCode;
    private String responseTitle;
    private String responseText;
    private Object responseObject;

    public ResultJSON(int responseCode, String responseTitle, String responseText)
    {
        this.responseCode = responseCode;
        this.responseTitle = responseTitle;
        this.responseText = responseText;
    }

}

File: 01 - Applications/futnetsinglemasterts/src/utils/beans/ResultJSON.ts
export default class ResultJSON {
  responseCode: Number
  responseTitle: string
  responseText: string
  responseObject: object

  constructor(
    responseCode: Number,
    responseTitle: string,
    responseText: string,
    responseObject: object
  ) {
    this.responseCode = responseCode
    this.responseTitle = responseTitle
    this.responseText = responseText
    this.responseObject = responseObject
  }
}
